code/modules/surgery/organs/_organ.dm

```diff
@@ -98,6 +103,15 @@ INITIALIZE_IMMEDIATE(/obj/item/organ)
 	receiver.organs_slot[slot] = src
 	owner = receiver
 
+	if(!IS_ROBOTIC_ORGAN(src) && (organ_flags & ORGAN_VIRGIN))
```

Comment:
The robotic organ checks feel arbitrary. Even if the organ itself does not have DNA, it has been inside someone... Someone who most likely *does* have DNA. And lots of blood.

Reply:
Nope nope nope.  A robotic organ does not have DNA since it is machinery.

Reply:
But if it got inserted in someone, it should get bloodied, and that blood will have the person's DNA. Am i misinterpreting something?

Reply:
If a robotic arm is amputated there shouldn't be blood.  Same logic.

Reply:
Also attaching blood to the organ, like how you are proposing, would result in double DNA when scanning.  Since the organ would have DNA attached + the blood with its own DNA.  Right now the blood decal is just fluff, since the DNA is attached to the organ directly.  Does that make sense?

Reply:
I get it now. You can close this.